### ISSUE TITLE
Mismatch in cdn and backend traffic

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#493]
I had a look at traffic incoming to the render servers and traffic leaving fastly headed to the backend. There's a significant mismatch.

![image](https://user-images.githubusercontent.com/1190866/103465153-11d48580-4cee-11eb-9a71-5fe2c9374787.png)
Upper is backend, lower is CDN.

`sum(rate(apache_sent_kilobytes_total{instance=~"(albi|bowser|necrosan|odin|pyrene|rhaegal|scorch|ysera)"}[$__rate_interval]))` and `sum(rate(fastly_rt_origin_fetch_resp_header_bytes_total{service_name=~"OSM Tiles CDN"}[$__rate_interval]))/1000 + sum(rate(fastly_rt_origin_fetch_resp_body_bytes_total{service_name=~"OSM Tiles CDN"}[$__rate_interval]))/1000` are graphs.

A similar difference is seen with the TPS

![image](https://user-images.githubusercontent.com/1190866/103465261-beaf0280-4cee-11eb-8056-86af33b4cc2e.png)

Some requests are still coming in from the old CDN, but at most that is 100/s.

I know it's possible that there are users bypassing the CDN and directly going to the backend, but I would be surprised if there's enough for 1.5k TPS.

**Comment by pnorman** [comment]
I did some digging on requests, looking at Fastly origin fetches (`sum(rate(fastly_rt_origin_fetches_total{service_name=~"OSM Tiles CDN"}[$__rate_interval]))`), Apache accesses (`sum(rate(apache_accesses_total{instance=~"(balerion|bowser|culebre|nidhogg|odin|pyrene|ysera)"}[$__rate_interval]))`), and modtile HTTP responses (`sum(rate(modtile_http_response_total{instance=~"(balerion|bowser|culebre|nidhogg|odin|pyrene|ysera)"}[$__rate_interval]))`)

apache requests = 2730 TPS
modtile responses = 2500 TPS.

This indicates significant non-tile traffic. Checking the logs for the same interval, there's 200 TPS getting 429 error codes for per-IP ratelimiting. This traffic wouldn't be reaching mod_tile, so explains that difference.

At the same time, fastly origin fetches = 1800.

Breaking down the mod_tile responses by status code gives me info.
HTTP 200 = 1750
HTTP 304 = 750
HTTP 404 = 0.1

It looks like fastly_rt_origin_fetches_total is only retrievals of tiles, not conditional gets with a 304. At the same time, there's still about 50TPS unaccounted for there. Some of that is going to be fastly health checks, but that's 1 request every 15 seconds per datacenter, or about 4TPS.

I also looked at other fastly metrics, and there's fastly_rt_origin_revalidations_total which is about 50 TPS, so that's probably it

However, I'm confused, because looking at the fastly RT API docs, origin_revalidations is supposed to be 304 status codes

So to summarize,

modtile responses = 200s + 304s + 404s + 5xxs
apache accesses = 200s + 304s + 404s + 429s+ 5xxs
fastly origin fetches + fastly origin revalidations = 200s

**Comment by grischard** [comment]
Is this still the case with [chef/#514](https://github.com/openstreetmap/chef/pull/514) done?

**Comment by grischard** [comment]
Yes, it is still the case.

![Image](https://user-images.githubusercontent.com/1556567/207927597-f9f98777-5b2e-445e-bee7-8e1237e79cba.png)

**Comment by grischard** [comment]
<img width="859" alt="image" src="https://user-images.githubusercontent.com/1556567/207933699-fb93bd12-fa6b-498b-ac5a-64d61473180d.png">
The delta is, on average, pretty stable. Going to assume this is monitoring.
